docs/docs/configuration.mdx

```diff
@@ -2,31 +2,81 @@
+| --http-port        | APPROZIUM_HTTP_PORT        | 6000       | Port for HTTP(S) API endpoints.                                                                                                                                                                                                                                                                                                                                           |
+| --grpc-port        | APPROZIUM_GRPC_PORT        | 6001       | Port for authenticator endpoint for clients.                                                                                                                                                                                                                                                                                                                              |
+
+## Logging Configuration
+
+| Name, shorthand    | Environment variable       | Default    | Description                                                                                                                                                                                                                                                                                                                                                               |
+|--------------------|----------------------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --log-level        | APPROZIUM_LOG_LEVEL        | info       | Supported selections are "trace", "debug", "info", "warn", "error", "fatal", and "panic". Upper case may be used.                                                                                                                                                                                                                                                         |
+| --log-format       | APPROZIUM_LOG_FORMAT       | text       | Also supports "json".                                                                                                                                                                                                                                                                                                                                                     |
```

Comment:
Can you please include `"text"` as a supported selection as well to match the above desc?

Reply:
Yep! https://github.com/cyralinc/approzium/pull/181/commits/bd9c01350393297b581861a0c9651774617bd329